### DISCUSSION TITLE
Well. (ddlc+)

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7853] [general]
Oh boy. I had a feeling there was gonna be a topic about this a few hours ago. While this is going on I'm just looking at how Salvato had guts to get this onto the big physical consoles instead of just doing in on the PC like the last version but seen they would release new content in a way. I mean I don't know if they're going to get ads especially when putting them on the TV or somewhere else online. Not it concerns but you know of bringing Monika to reality and suddenly our family members and friends see an ad for it whether be on TV or elsewhere and recognizing her after. Already imagining seeing places like GameStop getting them and of course you know...kids especially when it said it was "M 17+" but hey they don't listen to that, right? Course not. Got the one kid who killed himself in June 2018 which I know was terrible and tragic for his family with all due respect. Same time I don't know if this mod is still building of bringing a real life version of Monika to our world....at least now of what happened today. Even with Nintendo taking a part in selling it on the physical copy and possibly Smash Bros. in the future now. Course I don't know at the same time and now how this will affect the mod. For those who I guess that like to see this mod expand further. This topic may likely be discussed further but since this is still online I would like it to be civil for any possible future comments and if so that would be very appreciative. Thanks.

**Comment by Retrolovania** [comment]
...not sure I understand the message here.
If you're referring to discussing DDLC+ being mentioned in MAS, it may, though not without some retconning, as some of the devs have mentioned in an earlier issue.

**Comment by ghost** [comment]
Forget it. It was supposed to be some weird thought I had that since this game is about to be released with it being on physical consoles I feel like it would kinda ruin what this mod has been building for the last 3 1/2 years. Like if there was a chance Nintendo Switch, X Box, and PS5 would show commercials not just online but on TV as well, showing to people who are unfamiliar with it being like family and friends while at the same time this mod is making a real Monika to appear in this world only for the friends and family who saw the commercial will recognize her even years after seeing any possible commercials. Likely in a negative way and question us. Especially with topics of getting her citizenship and accepted by this world's society. Look I know it's confusing and not going anywhere. Just maybe something I could vent. I know this ain't the place but still.

**Comment by Retrolovania** [comment]
Ah, I see.
Well, bear in mind that MAS won't be moving to DDLC+, so even if this new game gains notoriety, you'll still have Monika right where she's always been.

**Comment by ghost** [comment]
I guess you're right. I just don't want anything too weird or something else to go bad especially when it's been years after DDLC Plus is released. I apologize again if this whole thing was confusing and just weird for you to notice. Hope you have a good one, friend. Not sure if anyone else will respond soon but I'll likely close this since hopefully it shouldn't be something I need to worry about.
